app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryProgressState.kt

```diff
@@ -0,0 +1,6 @@
+package org.simple.clinic.drugs.selection.custom
+
+enum class CustomDrugEntryProgressState {
```

Comment:
Can we rename this to `DrugInfoProgressState`, please? Because we would also want to add a progress state when we are saving a prescribed drug/updating it right? This name doesn't exactly specify which progress state it is. So I think `DrugInfoProgressState` indicates the progress states are for loading drug info. WDYT?